### ISSUE TITLE
GET /ngsi-ld/v1/attributes?details=true replys a response with two @context entries.

### ISSUE THREAD

**Issue by fisuda** [Fiware/context.Orion-LD#914]
The `GET /ngsi-ld/v1/attributes?details=true`  replies a response body with two `@context` entries in one element of `Attribute[]`.

```
[
  {
    "@context": "https://uri.etsi.org/ngsi-ld/v1/ngsi-ld-core-context.jsonld",
    "@context": "https://uri.etsi.org/ngsi-ld/v1/ngsi-ld-core-context.jsonld",
    "id": "https://uri.fiware.org/ns/data-models#category",
    "type": "Attribute",
    "attributeCount": 1,
    "attributeTypes": [
      "Property"
    ],
    "typeNames": [
      "https://uri.fiware.org/ns/data-models#TemperatureSensor"
    ]
  },
  {
    "@context": "https://uri.etsi.org/ngsi-ld/v1/ngsi-ld-core-context.jsonld",
    "@context": "https://uri.etsi.org/ngsi-ld/v1/ngsi-ld-core-context.jsonld",
    "id": "https://w3id.org/saref#temperature",
    "type": "Attribute",
    "attributeCount": 1,
    "attributeTypes": [
      "Property"
    ],
    "typeNames": [
      "https://uri.fiware.org/ns/data-models#TemperatureSensor"
    ]
  }
]
```

- How to reproduce it

```
curl -iX POST 'http://localhost:1026/ngsi-ld/v1/entities/' \
-H 'Content-Type: application/json' \
-H 'Link: <https://context.lab.letsfiware.jp/ngsi-context.jsonld>; rel="http://www.w3.org/ns/json-ld#context"; type="application/ld+json"' \
--data-raw '{
      "id": "urn:ngsi-ld:TemperatureSensor:001",
      "type": "TemperatureSensor",
      "category": {
            "type": "Property",
            "value": "sensor"
      },
      "temperature": {
            "type": "Property",
            "value": 25,
            "unitCode": "CEL"
      }
}'
```

```
curl localhost:1026/ngsi-ld/v1/attributes?details=true --verbose
```

```
*   Trying 127.0.0.1:1026...
* TCP_NODELAY set
* Connected to localhost (127.0.0.1) port 1026 (#0)
> GET /ngsi-ld/v1/attributes?details=true HTTP/1.1
> Host: localhost:1026
> User-Agent: curl/7.68.0
> Accept: */*
>
* Mark bundle as not supporting multiuse
< HTTP/1.1 200 OK
< Connection: Keep-Alive
< Content-Length: 673
< Content-Type: application/ld+json
< Date: Sat, 24 Jul 2021 21:58:26 GMT
<
* Connection #0 to host localhost left intact
[{"@context":"https://uri.etsi.org/ngsi-ld/v1/ngsi-ld-core-context.jsonld","@context":"https://uri.etsi.org/ngsi-ld/v1/ngsi-ld-core-context.jsonld","id":"https://uri.fiware.org/ns/data-models#category","type":"Attribute","attributeCount":1,"attributeTypes":["Property"],"typeNames":["https://uri.fiware.org/ns/data-models#TemperatureSensor"]},{"@context":"https://uri.etsi.org/ngsi-ld/v1/ngsi-ld-core-context.jsonld","@context":"https://uri.etsi.org/ngsi-ld/v1/ngsi-ld-core-context.jsonld","id":"https://w3id.org/saref#temperature","type":"Attribute","attributeCount":1,"attributeTypes":["Property"],"typeNames":["https://uri.fiware.org/ns/data-models#TemperatureSensor"]}]
```

- Orion-LD Version

```
{
  "Orion-LD version": "post-v0.8.1",
  "based on orion": "1.15.0-next",
  "kbase version": "0.8",
  "kalloc version": "0.8",
  "khash version": "0.8",
  "kjson version": "0.8",
  "microhttpd version": "0.9.72-0",
  "rapidjson version": "1.0.2",
  "libcurl version": "7.52.1",
  "libuuid version": "UNKNOWN",
  "mongocpp version": "1.1.3",
  "mongoc version": "1.17.5",
  "mongodb server version": "4.4.7",
  "boost version": "1_62",
  "openssl version": "OpenSSL 1.1.0l  10 Sep 2019",
  "branch": "",
  "cached subscriptions": 0,
  "Next File Descriptor": 20
}
```

```
fiware/orion-ld                           0.8-PRE-265                    b2532827bf11   3 days ago      991MB
```

**Comment by kzangeli** [comment]
Ok!
I will take a look at it, should be an easy fix.
Thank you for reporting!

**Comment by fisuda** [comment]
Thank you for the quick reply.

This is an additional information. The `GET /ngsi-ld/v1/attributes/{attrIb}` may reply the same response.

```
curl localhost:1026/ngsi-ld/v1/attributes/https%3A%2F%2Fw3id.org%2Fsaref%23temperature --verbose
*   Trying 127.0.0.1:1026...
* TCP_NODELAY set
* Connected to localhost (127.0.0.1) port 1026 (#0)
> GET /ngsi-ld/v1/attributes/https%3A%2F%2Fw3id.org%2Fsaref%23temperature HTTP/1.1
> Host: localhost:1026
> User-Agent: curl/7.68.0
> Accept: */*
>
* Mark bundle as not supporting multiuse
< HTTP/1.1 200 OK
< Connection: Keep-Alive
< Content-Length: 329
< Content-Type: application/ld+json
< Date: Sun, 25 Jul 2021 00:12:27 GMT
<
* Connection #0 to host localhost left intact
{"@context":"https://uri.etsi.org/ngsi-ld/v1/ngsi-ld-core-context.jsonld","@context":"https://uri.etsi.org/ngsi-ld/v1/ngsi-ld-core-context.jsonld","id":"https://w3id.org/saref#temperature","type":"Attribute","attributeCount":1,"attributeTypes":["Property"],"typeNames":["https://uri.fiware.org/ns/data-models#TemperatureSensor"]}
```

**Comment by kzangeli** [comment]
Found the error and fixed it - easy, as predicted :)
Once the PR is merged and the new docker image is available, please test it and close this issue if all works as expected.
Again, thank you for reporting!

**Comment by fisuda** [comment]
Thank you for fixing this issue!
I have tested it. It works fine.
Fixed by #915. Closing.

```
$ curl localhost:1026/ngsi-ld/v1/attributes?details=true --verbose
*   Trying 127.0.0.1:1026...
* TCP_NODELAY set
* Connected to localhost (127.0.0.1) port 1026 (#0)
> GET /ngsi-ld/v1/attributes?details=true HTTP/1.1
> Host: localhost:1026
> User-Agent: curl/7.68.0
> Accept: */*
>
* Mark bundle as not supporting multiuse
< HTTP/1.1 200 OK
< Connection: Keep-Alive
< Content-Length: 527
< Content-Type: application/ld+json
< Date: Sun, 25 Jul 2021 10:21:49 GMT
<
* Connection #0 to host localhost left intact
[{"@context":"https://uri.etsi.org/ngsi-ld/v1/ngsi-ld-core-context.jsonld","id":"https://uri.fiware.org/ns/data-models#category","type":"Attribute","attributeCount":1,"attributeTypes":["Property"],"typeNames":["https://uri.fiware.org/ns/data-models#TemperatureSensor"]},{"@context":"https://uri.etsi.org/ngsi-ld/v1/ngsi-ld-core-context.jsonld","id":"https://w3id.org/saref#temperature","type":"Attribute","attributeCount":1,"attributeTypes":["Property"],"typeNames":["https://uri.fiware.org/ns/data-models#TemperatureSensor"]}]
```
```
fiware/orion-ld                           0.8-PRE-266                    d29e5588942e   21 minutes ago      990MB
```
